### PR TITLE
Cleanup/getting started/cli

### PR DESCRIPTION
**Pull request by DavidSouther**
1. Consistently report Bedrock and other engine errors.
2. Better align CLI logging and output at the right information level.
3. Fix several bugs from streaming handling.
4. Rework defaults for better getting started experience

### CODE REVIEW

File: core/src/content/template.ts [hunk 1]
```diff
@@ -63,3 +65,27 @@ export const GLOBAL_VIEW: View = {
...
+/**
+ * Read, parse, and validate a template view.
+ */
+
+export async function loadTemplateView(
+  fs: FileSystem,
+  ...paths: string[]
+): Promise<View> {
+  if (!paths) return {};
+  let view = /* @type View */ {};
```
Comment (cpyle0819) [inline core/src/content/template.ts:77]: TS cast instead of JS doc here?

File: core/src/engine/bedrock/bedrock.ts [hunk 2]
```diff
@@ -32,22 +32,29 @@ const MODEL_MAP: Record<string, string> = {
...
+  "sonnet:48k": "anthropic.claude-3-sonnet-20240229-v1:0:48k",
+  "haiku:48k": "anthropic.claude-3-haiku-20240307-v1:0:48k",
+  "opus:48k": "anthropic.claude-3-opus-20240229-v1:0:48k",
+  "sonnet:200k": "anthropic.claude-3-sonnet-20240229-v1:0:200k",
+  "haiku:200k": "anthropic.claude-3-haiku-20240307-v1:0:200k",
+  "opus:200k": "anthropic.claude-3-opus-20240229-v1:0:200k",
 };
 
+declare module "@davidsouther/jiffies/lib/cjs/log.js" {
+  interface Logger {
```
Comment (cpyle0819) [inline core/src/engine/bedrock/bedrock.ts:44]: How/where is this being used?

File: core/src/content/template.ts (hunk 1 above)
Comment (DavidSouther) [reply core/src/content/template.ts:77]: Done.

File: core/src/engine/bedrock/bedrock.ts (hunk 2 above)
Comment (DavidSouther) [reply core/src/engine/bedrock/bedrock.ts:44]: Done.

**Review by cpyle0819 — APPROVED** [review]

**Pull request merged**